src/navigation/card/screens/settings/Referral.tsx

```diff
@@ -184,7 +184,7 @@ const Referral = ({}) => {
 
   const getStatus = (status: string, expiration: number) => {
     return status === 'pending' && currentDate >= expiration
-      ? 'Expired'
+      ? t('Expired')
       : status.charAt(0).toUpperCase() + status.slice(1);
```

Comment:
Is status enumerated somewhere? If so we should create a map and map each known status to a translated value